apidoc/Titanium/UI/WebViewConfiguration.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: Titanium.UI.WebViewConfiguration
+summary: A collection of properties used to initialize a web view.
+description: |
+    Use the <Titanium.UI.createWebViewConfiguration> method to create. Use it as parameter of <Titanium.UI.WebView.configuration>
+    Using this you can determine how soon a webpage is rendered, how media playback is handled, the granularity of items that the 
+    user can select, and many other options. This property can only be set when creating the webview and will be ignored when set afterwards.
+
+    E.g.
+
+    var config = Ti.UI.createWebViewConfiguration({
+        allowsPictureInPictureMediaPlaback: true
+    });
+
+    var webView = WK.createWebView({
+        configuration: config
+    });
```

Comment:
The `url` property is always required. Did you test this example code? We should always test code before putting it into the docs, since some docs expect copy and paste examples to just work.